Model/AbstractRefreshToken.php

```diff
@@ -56,7 +56,7 @@ public static function createForUserWithTtl(string $refreshToken, UserInterface
      */
     public function __toString()
     {
-        return $this->getRefreshToken();
+        return $this->getRefreshToken() ?: '';
```

Comment:
`getRefreshToken()` has a nullable return, `__toString()` should always be a string.